src/app/helpers/telemetryHelper.js

```diff
@@ -54,7 +54,7 @@ module.exports = {
     context.did = req.session.deviceId
     context.rollup = telemetry.getRollUpData(dims)
     const actor = telemetry.getActorData(req.session.userId, 'user')
-   console.log('logging session start event', context.did);
+    console.log('logging session start event', context.did);
```

Comment:
we have logger instead of using console.log we can use 
logger.info({ msg: 'string' });

Reply:
this one I haven't add, this line is already there, only lint errors are fixed.